tasks/system_probe.py

```diff
@@ -38,6 +38,7 @@
 KITCHEN_ARTIFACT_DIR = os.path.join(KITCHEN_DIR, "site-cookbooks", "dd-system-probe-check", "files", "default", "tests")
 TEST_PACKAGES_LIST = ["./pkg/ebpf/...", "./pkg/network/..."]
 TEST_PACKAGES = " ".join(TEST_PACKAGES_LIST)
+is_windows = os.name == "nt"
```

Comment:
In other places, we check for windows with `sys.platform == "win32"`. Should we use that instead?

Reply:
Nice catch, I think they're equivalent, but I'll change it to be consistent.